### ISSUE TITLE
Update?

### ISSUE THREAD

**Issue by roast247** [mavenboard/flipperesl#1]
Is this still in development? downloaded it from flipc.org but nothing on the addon is working.

**Comment by mavenboard** [comment]
yes, it's still in development, and it's going to be for a long time join
flipper xtreme discord there are more information there in the bounty
channel
